presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -385,6 +386,16 @@ void PrestoServer::run() {
                 http::kMimeTypeApplicationJson)
             .sendWithEOM();
       });
+  if (systemConfig->prestoNativeSidecar()) {
+    httpServer_->registerGet(
```

Comment:
We have another endpoint that is for the sidecar (the session one). If we keep adding them like this in various places. I wonder if it would make sense to have a function that registers all the sidecar endpoints in a single place. All that is needed for these so far is that the httpServer_ is around (the session one relies on taskManager_). Perhaps a function `registerSidecarEndpoints()` that is a method of prestoServer called prior to calling the PerodicTaskManager start where everything needs to have been set up?
There are going to be a few more endpoints (for the eval). Would this grouping make more sense?

Reply:
+1. Agree. Lets consolidate all the sidecar endpoints in a single function,

Reply:
Updated, thanks!